torch/nn/quantized/modules/utils.py

```diff
@@ -4,7 +4,9 @@
 from torch.nn.modules.module import _addindent
 
 def _quantize_weight(float_wt, observer):
-    wt_scale, wt_zp = observer.calculate_qparams()
+    device = float_wt.device
+    wt_scale, wt_zp = observer.calculate_qparams()  # may be better to make calculate qparams give on correct device
```

Comment:
can we make calculate_qparams to return scale/zero_point in correct device?

Reply:
yeah, agreed with the comment

Reply:
this change is in the most recent version